nodejs/week3/homework/warmup/index.js

```diff
@@ -0,0 +1,32 @@
+const express = require("express");
+const app = express();
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+
+
+app.get("/calculator/add", (req, res) => {
+    const firstNumber = Number(req.query.first);
+    const secondNumber = Number(req.query.second);
+    res.send(`${firstNumber + secondNumber}`);
+  });
```

Comment:
What if one of the parameters (or both) is not a valid number? The same problem with subtract, multiply and div below.

Can you fix that?

Reply:
Done